packages/safe-tools-client/app/components/schedule-collapse-panel/index.css

```diff
@@ -0,0 +1,136 @@
+.collapse-panel {
+  --collapse-panel-border-radius: 20px;
+  --collapse-panel-summary-height: 80px;
+  --collapse-panel-summary-anim-bottom-margin: 8px;
+
+  background: url('/images/backgrounds/schedule-panel.svg') no-repeat bottom/cover white;
+  border-radius: var(--collapse-panel-border-radius);
+  overflow: hidden;
+}
+
+.collapse-panel__summary {
+  display: flex;
+  flex: 1;
+  height: var(--collapse-panel-summary-height);
+  padding: 0 var(--boxel-sp-xl);
+  border-radius: var(--collapse-panel-border-radius);
+  font-size: var(--boxel-font-size);
+  transition: margin-bottom var(--boxel-transition);
+}
+
+/* Adds margin to fake animation */
+.collapse-panel[open] .collapse-panel__summary {
+  margin-bottom: var(--collapse-panel-summary-anim-bottom-margin);
+}
+
+.collapse-panel__summary-collapsed {
+  display: inherit;
```

Comment:
We need to specify `display: flex; width: 100%;` explicitly because Safari is fussy.